test/run_test.go

```diff
@@ -14,10 +16,30 @@ import (
 const minimalPkg = "minimalpkg"
 
 func TestAutogeneratedNoIssues(t *testing.T) {
+	_, ci := os.LookupEnv("CI")
+	if runtime.GOOS == "windows" && ci {
+		// Tests on Windows and GitHub action that use a file produce a warning and so an exit code 2:
+		// level=warning msg=\"[config_reader] Can't pretty print config file path: can't get relative path for path C:\\\\Users\\\\runneradmin\\\\AppData\\\\Local\\\\Temp\\\\golangci_lint_test3234185281.yml and root D:\\\\a\\\\golangci-lint\\\\golangci-lint\\\\test: Rel: can't make C:\\\\Users\\\\runneradmin\\\\AppData\\\\Local\\\\Temp\\\\golangci_lint_test3234185281.yml relative to D:\\\\a\\\\golangci-lint\\\\golangci-lint\\\\test\"
+		//
+		// In the context of a test that ExpectNoIssues this is problem.
+		//
+		// NOTE(ldez): I don't want to create flags only for running tests on Windows + GitHub Action.
+		t.Skip("on Windows + GitHub Action")
+	}
```

Comment:
I propose to move these lines to a helper function to avoid duplication.

Reply:
we will do that if we have more cases.